scripts/fast-reboot

```diff
@@ -1,9 +1,13 @@
 #!/bin/bash
 
+REBOOT_USER=$(logname)
+REBOOT_TIME=$(date)
+REBOOT_CAUSE_FILE="/var/cache/sonic/reboot-cause.txt"
+
 # Check root privileges
 if [[ "$EUID" -ne 0 ]]
 then
-  echo "Please run as root"
+  echo "This command must be run as root" >&2
   exit
```

Comment:
>exit [](start = 2, length = 4)

Not your code, but I suggest to exit with a unique non-zero code to indicate error